### ISSUE TITLE
scrolling ins hardware accelerated

### ISSUE THREAD

**Issue by dustingetz** [seatgeek/react-infinite#63]

**Comment by dustingetz** [comment]
I am attempting to address this in my fork - if it works, this fix can probably be back-ported.

**Comment by Polyrhythm** [comment]
:+1:. I think `requestAnimationFrame` might be the way to go here.

**Comment by diffractometer** [comment]
@dustingetz did you make a pull request? I would be interested in checking it out.

**Comment by dustingetz** [comment]
I haven't had time to submit a PR to this project, but you can see what I did in my fork here: https://github.com/dustingetz/react-infinite/blob/master/src/react-infinite2.jsx#L93

**Comment by dustingetz** [comment]
I will eventually submit a PR to this project, its just may be a month or two.

**Comment by diffractometer** [comment]
Cool, thanks for the update. I will keep one eye on the project!

**Comment by garetht** [comment]
I'd really like to get this in as well. We're a little busy at the moment, but I'll try and get some time to do it soon.

**Comment by webyak** [comment]
+1

**Comment by jasongonzales23** [comment]
ugh, this is almost a year old, no progress?

**Comment by diffractometer** [comment]
@jasongonzales23 submit a PR if it's important enough to lament with an 'ugh' brah.

**Comment by jasongonzales23** [comment]
Fair enough, apologies, was just having a very frustrating day at the time. If you're in SF I'd be happy to buy you a beer or two to make it up to you 🍻

**Comment by diffractometer** [comment]
@jasongonzales23 No worries. I'm just a troll who uses this project, not the author and I've been there. If you're in Chicago I will buy you a beer.

**Comment by jasongonzales23** [comment]
See, we aren't a country divided afterall. Take that politicians...
